packages/pluggableWidgets/datagrid-web/src/utils/__tests__/settings.spec.ts

```diff
@@ -0,0 +1,209 @@
+import { useSettings } from "../settings";
+import { ColumnWidth, TableColumn } from "../../components/Table";
+import { EditableValueBuilder } from "@native-mobile-resources/util-widgets";
+import { HidableEnum } from "../../../typings/DatagridProps";
+import { renderHook } from "@testing-library/react-hooks";
+
+describe("useSettings Hook", () => {
```

Comment:
Very nice separation of testing the hook behaviour!

Reply:
Thx :)